addons/website/static/src/scss/website.ui.scss

```diff
@@ -41,6 +41,36 @@ body.o_connected_user {
     }
 }
 
+//FULLSCREEN INDICATION LAYOUT
+.o_fullscreen_indication {
+    display: flex;
+    position: absolute;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    margin-top: $o-navbar-height!important;
+    opacity: 0;
+    p {
+        padding: 15px 30px;
+        background-color: rgba(0, 0, 0, 0.7);
+        color: white;
+        span {
+            border: solid 1px;
+            border-color: white;
+            border-radius: 5px;
+            padding: 7px 5px;
+            margin: 5px;
+        }
+    }
+    &.transition {
+        transition: opacity 400ms ease 0s;
+    }
+    &.visible {
```

Comment:
Use prefixed class names instead (`.o_`) to avoid having external CSS mess with your layout/design